### PR TITLE
FlowRun.uuid part 2 

### PR DESCRIPTION
**Pull request by rowanseymour**

### CODE REVIEW

File: temba/flows/migrations/0108_populate_flowrun_uuid.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    run_ids = list(FlowRun.objects.filter(uuid=None).values_list('id', flat=True))
+    if not run_ids:
+        return
+
+    print("Fetched %d flow run ids that need UUIDs..." % len(run_ids))
+    num_updated = 0
+
+    for run_ids in run_ids:
+        FlowRun.objects.filter(id=run_ids).update(uuid=uuid4())
+        num_updated += 1
```
Comment (ericnewcomer) [inline temba/flows/migrations/0108_populate_flowrun_uuid.py]: for **run_id** in run_ids..

Same deal for id=run_id

Comment (rowanseymour) [inline temba/flows/migrations/0108_populate_flowrun_uuid.py]: Fixed!

File: temba/flows/migrations/0108_populate_flowrun_uuid.py [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+def populate_flowrun_uuid(FlowRun):
+    run_ids = list(FlowRun.objects.filter(uuid=None).values_list('id', flat=True))
+    if not run_ids:
+        return
+
+    print("Fetched %d flow run ids that need UUIDs..." % len(run_ids))
+    num_updated = 0
+
+    for run_id in run_ids:
+        FlowRun.objects.filter(id=run_id).update(uuid=uuid4())
```
Comment (nicpottier) [inline temba/flows/migrations/0108_populate_flowrun_uuid.py]: No way to do this as a batch huh?

Comment (rowanseymour) [inline temba/flows/migrations/0108_populate_flowrun_uuid.py]: See #1346

**Pull request merged**